src/System.Management.Automation/engine/hostifaces/RunspacePoolInternal.cs

```diff
@@ -76,25 +72,19 @@ internal class RunspacePoolInternal
         /// </exception>
         public RunspacePoolInternal(int minRunspaces,
                 int maxRunspaces,
-                RunspaceConfiguration runspaceConfiguration,
                 PSHost host)
             : this(minRunspaces, maxRunspaces)
         {
-            if (runspaceConfiguration == null)
-            {
-                throw PSTraceSource.NewArgumentNullException("runspaceConfiguration");
-            }
-
             if (host == null)
             {
                 throw PSTraceSource.NewArgumentNullException("host");
             }
 
-            rsConfig = runspaceConfiguration;
             this.host = host;
             pool = new Stack<Runspace>();
             runspaceRequestQueue = new Queue<GetRunspaceAsyncResult>();
             ultimateRequestQueue = new Queue<GetRunspaceAsyncResult>();
+            _initialSessionState = InitialSessionState.CreateDefault2();
```

Comment:
Same here. If we want to mimic `RunspaceConfiguration`, then `CreateDefault()` should be used.

Reply:
will change